server/data/prisonRegisterClient.test.ts

```diff
@@ -58,7 +58,7 @@ describe('prisonRegisterClient', () => {
         userMessage: 'An unexpected error occurred',
         developerMessage: 'An unexpected error occurred',
       }
-      prisonRegisterApi.get('/prisons').reply(500, expectedResponseBody)
+      prisonRegisterApi.get('/prisons').thrice().reply(500, expectedResponseBody)
```

Comment:
On error `RestClient` will now retry `GET` requests twice for a total call count of three. All three API calls need to return a 500 error for this test to pass.